src/commands/moderation/kick.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  highestRole,
+  higherRolePosition,
+  Member,
+  botID,
+  sendDirectMessage,
+} from "../../../deps.ts";
+import { botCache } from "../../../mod.ts";
+import { createCommandAliases } from "../../utils/helpers.ts";
+import { PermissionLevels } from "../../types/commands.ts";
+import { kick } from "https://x.nest.land/Discordeno@8.4.4/src/handlers/member.ts";
+
+botCache.commands.set(`kick`, {
+  name: `kick`,
+  permissionLevels: [PermissionLevels.MODERATOR, PermissionLevels.ADMIN],
+  botServerPermissions: ["KICK_MEMBERS"],
+  arguments: [
+    { name: "member", type: "member", required: false },
```

Comment:
```suggestion
    { name: "member", type: "member" },
```
Member should never be optional in kick.